packages/xo-web/src/xo-app/host/action-bar.js

```diff
@@ -10,11 +10,19 @@ import {
   stopHost,
 } from 'xo'
 
+const _restartHostAgent = ({ host, fetchStats }) => {
+  //  Don't fetch host stats because there is no connection to the host during reboot.
```

Comment:
After testing the restart of the toolstack on a virtual XCP NG, the host doesn't reboot. 

There is a special case when it reboots?

Reply:
Basically, when we restart toolstack, we restart all xapi services according to this [script](https://github.com/xapi-project/xen-api/blob/master/scripts/xe-toolstack-restart). Maybe the commentary is badly formed, I will fix it.

Reply:
Yes, `restart` instead of `reboot` is a good idea :+1: